1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -490,3 +490,44 @@ const quotes = [
     author: "Zig Ziglar",
   },
 ];
+
+//variables
+const quoteP = document.querySelector("#quote"),
+  authorP = document.querySelector("#author"),
+  newQuoteBtn = document.querySelector("#new-quote"),
+  autoGenerate = document.querySelector(".checkbox");
+let intervalID;
+
+// eventListeners
+eventlisteners();
+function eventlisteners() {
+  // Get a quote on load
+  document.addEventListener("DOMContentLoaded", showQuoteOnLoad);
+  // Get a quote on click
+  newQuoteBtn.addEventListener("click", showQuoteOnClick);
+  autoGenerate.addEventListener("click", showAuto);
+}
+
+//functions
+// show quote on load
+function showQuoteOnLoad() {
+  const quoteOnLoad = pickFromArray(quotes);
+  quoteP.textContent = quoteOnLoad.quote;
+  authorP.textContent = `- ${quoteOnLoad.author}`;
+}
+// show quote on click
+function showQuoteOnClick() {
+  const quoteOnLoad = pickFromArray(quotes);
+  quoteP.textContent = quoteOnLoad.quote;
+  authorP.textContent = `- ${quoteOnLoad.author}`;
+}
+// show quote automaticlly
+function showAuto() {
+  if (autoGenerate.checked) {
+    intervalID = setInterval(() => {
+      showQuoteOnClick();
```

Comment:
great job to make it as function and reuse it!

Reply:
🙌